README.md

```diff
@@ -167,6 +167,8 @@
   - <sub>ISBN 13: 9781449328016</sub> 
 - [Database Systems: Design, Implementation, and Management](https://books.google.com/books/about/Database_Systems_Design_Implementation_M.html?id=elVZCwAAQBAJ&printsec=frontcover&source=kp_read_button)
   - <sub>ISBN 13: 9781337627900</sub> 
+- [Readings in Database Systems](https://www.amazon.com/Readings-Database-Systems-MIT-Press/dp/0262693143)
+  - <sub>ISBN-13: 978-0262693141</sub> 
```

Comment:
For more cohesion, remove the `-` ont the `ISBN 13`

Reply:
@Farnots thanks for the suggestion but I check for other books, some of them have '-' and some of them don't.